pinot-broker/src/main/java/org/apache/pinot/broker/broker/helix/ClusterChangeMediator.java

```diff
@@ -116,11 +117,18 @@ public void run() {
     };
   }
 
-  private void processClusterChange(ChangeType changeType, ClusterChangeHandler changeHandler) {
+  private void processClusterChange(ChangeType changeType, List<ClusterChangeHandler> changeHandlers) {
     long startTime = System.currentTimeMillis();
     LOGGER.info("Start processing {} change", changeType);
-    changeHandler.processClusterChange();
-    long endTime = System.currentTimeMillis();
+    long handlerStartTime = startTime;
+    for (ClusterChangeHandler changeHandler : changeHandlers) {
+      changeHandler.processClusterChange(changeType);
```

Comment:
It might be good to handle exceptions from this handler here so we don't skip others if one has an error.

Reply:
Done